build: Add .bzl/BUILD files to @angular/bazel package

alexeagle opened the pull request:
It's now possible to use it as a "hybrid" package in latest rules_nodejs, so no WORKSPACE dep is needed

kyliau commented on packages/bazel/package.json:
```diff
@@ -35,8 +41,5 @@
     "packageGroup": "NG_UPDATE_PACKAGE_GROUP"
-  },
-  "scripts": {
-    "postinstall": "node ./check_version.js"
```
Should `check_version.js` be removed as well?

alexeagle replied:
Done

gregmagolan commented on packages/bazel/BUILD.bazel:
```diff
@@ -1,23 +1,41 @@
+    replacements = {
+        "(#|\/\/)\\s+BEGIN-DEV-ONLY[\\w\W]+?(#|\/\/)\\s+END-DEV-ONLY": "",
+        "//packages/bazel/": "//",
+        "@ngdeps//": "@npm//",
```
We'll be able to remove @ngdeps now and just use @npm for the angular repo. This could be done in a follow up PR as it would be a big search & replace.

alexeagle committed:
build: Add .bzl/BUILD files to @angular/bazel package

It's now possible to use it as a hybrid package in latest rules_nodejs, so no WORKSPACE dep is needed
